libflux/flux-core/src/doc/mod.rs

```diff
@@ -1294,6 +1299,77 @@ mod test {
         );
     }
     #[test]
+    fn test_value_doc_multiline_headline_no_desc() {
+        let src = "
+        // Package foo does a thing.
+        package foo
+
+        // a is a constant. This headline has `code`
+        // and multiple lines.
+        a = 1
+        ";
+        let loc = Locator::new(&src[..]);
+        assert_docs_full(
+            src,
+            PackageDoc {
+                path: "path".to_string(),
+                name: "foo".to_string(),
+                headline: "Package foo does a thing.".to_string(),
+                description: None,
+                members: map![
+                    "a" => Doc::Value(Box::new(ValueDoc{
+                        name: "a".to_string(),
+                        headline: "a is a constant. This headline has `code`\nand multiple lines.".to_string(),
+                        description: None,
+                        flux_type: "int".to_string(),
+                        is_option: false,
+                        source_location: loc.get(7,9,7,14),
+                        examples: vec![],
+                        metadata: None,
+                    })),
+                ],
+                examples: Vec::new(),
+                metadata: None,
+            },
+            vec![],
+        );
+    }
+    #[test]
+    fn test_value_doc_code_headline_no_desc() {
```

Comment:
same thing here, the test passed without changes to the parser, but I wanted to validate that the normal headline didn't have the same problem as the headline for a parameter within a list item.